com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -0,0 +1,198 @@
+using MLAPI.Serialization.Pooled;
+using MLAPI.Serialization;
+using MLAPI.Configuration;
+using MLAPI.Profiling;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using System.IO;
+
+namespace MLAPI
+{
+    class MessageBatcher
+    {
+        public class SendStream
```

Comment:
Is this a class vs. a struct so it's a ref type not a value type?

Reply:
I believe I wanted a ref type. What would be the upside of having a struct here?